superbench/benchmarks/micro_benchmarks/directx_mem_bw_performance/Main.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include <sstream>
+
+#include "GPUMemRwBw.h"
+
+int main(int argc, char *argv[]) {
+    std::unique_ptr<BenchmarkOptions> opts = std::make_unique<BenchmarkOptions>(argc, argv);
```

Comment:
>std::unique_ptr<BenchmarkOptions> opts = std::make_unique<BenchmarkOptions>(argc, argv); [](http://example.com/codeflow?start=4&length=88)

I guess you can use stack to save these variables as well like:

BenchmarkOptions option(argc, argv);
option.init();

GPUMemRwBw benchmark(&option);
benchmark.Run();

Reply:
thanks, updated